si.tano.Vremenar.yaml

```
@@ -0,0 +1,59 @@
+---
+app-id: si.tano.Vremenar
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: vremenar
+rename-icon: vremenar
+rename-desktop-file: vremenar.desktop
+copy-icon: true
+finish-args:
+  # Required due to being a GUI application
+  - "--socket=x11"
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - "--share=ipc"
+  # Required for experimental wayland support
+  - "--socket=wayland"
+  # Required for OpenGL
+  - "--device=dri"
+  # As a weather application, networking is required
+  - "--share=network"
+  # Required for notifications in various desktop environments
+  - "--talk-name=org.freedesktop.Notifications"
+  - "--talk-name=org.kde.StatusNotifierWatcher"
+  # Required due to KDE's special way of handling notifications
+  - "--own-name=org.kde.StatusNotifierItem-2-1"
```

```suggestion
  - "--own-name=org.kde.*"
```
The `2-1` part is unstable and may change depending on the number of Qt applications trying to set up a tray icon.